app/src/main/java/org/simple/clinic/user/LoggedInUser.kt

```diff
@@ -59,9 +59,12 @@ data class LoggedInUser(
   @Dao
   interface RoomDao {
 
-    @Query("SELECT * FROM LoggedInUser")
+    @Query("SELECT * FROM LoggedInUser LIMIT 1")
     fun user(): Flowable<List<LoggedInUser>>
```

Comment:
Why have a limit when we can only have one row in this table ever?

Reply:
No specific reason. I'd say it tells me at a glance when I read the SQL statement that I expect only one row to be present.